frontend/src/app/testResults/uploads/CsvSchemaDocumentation.tsx

```diff
@@ -160,7 +160,7 @@ const CsvSchemaDocumentation = () => {
           <ul>
             <li>
               <a
-                href="/getting-started/testing-facilities/csv-schema"
```

Comment:
This link led to an error page since that page doesn't exisit

Reply:
this change can close #4225 as well